app-modules/ai/tests/Feature/Filament/Pages/ManageAiSettingsPageTest.php

```diff
@@ -63,17 +63,6 @@
         ->assertStatus(403);
 });
 
-it('loads if you have the correct access to ai settings', function () {
-    $user = User::factory()->licensed(LicenseType::ConversationalAi)->create();
-
-    $user->givePermissionTo(['product_admin.view-any']);
-
-    actingAs($user);
```

Comment:
I think you can restore this test. But use the `asSuperAdmin` helper as a replacement for all 3 of these which would set the right license and the correct role.